database.py

```diff
@@ -0,0 +1,45 @@
+from peewee import *
+
+
+cippe_db = SqliteDatabase("cippedb.db", autorollback=True)
+
+class BaseModel(Model):
+    class Meta:
+        database = cippe_db
+
+
+class Recipie(BaseModel):
+    id = AutoField()
+    food_name = TextField(unique=True)
+    description =TextField()
+    top_image = TextField()
+    bottom_image = TextField()
+    class Meta:
+        table_name = "recipies"
+
+class Ingredient(BaseModel):
+    id = AutoField()
+    food_id = ForeignKeyField(Recipie, backref="ingredients")
+    category = TextField()
+    name = TextField()
+    quantity = TextField()
+    Type = TextField()
```

Comment:
why is `"Type"` in uppercase?

Reply:
python has a built in type for checking whether a variable is a dict or list or string etc

Reply:
Better to change the name then, so your naming convention is consistent. Maybe something like `ingredient_type` or something else.

Reply:
ok then